tests/wpunit/SSP_Episode_Functions_Test.php

```
@@ -0,0 +1,336 @@
+		$excerpt = ssp_get_episode_excerpt( 99999 );
+
+		$this->assertIsString( $excerpt );
+	}
+
+	/**
+	 * @covers ssp_episode_image()
+	 */
+	public function test_ssp_episode_image() {
+		$this->assertTrue( function_exists( 'ssp_episode_image' ) );
+
+		$image = ssp_episode_image( $this->test_episode_id );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$image'. (Unused Code Rules)

(UnusedLocalVariable)